### ISSUE TITLE
Problems attaching files in mail

### ISSUE THREAD

**Issue by nsabatelli** [roundcube/roundcubemail#7660]
Hello,
I have the following problem when sending an email.
If I attach a file to the email before entering the recipient data and the email text, the email is sent without attachment.
On the other hand, if I enter the recipient's data, the text of the e-mail and then attach a file to the e-mail, it is sent correctly with the attachment.

In my implementation I use **Roundcube version 1.4.9** with the following plugins: _automatic_addressbook, calendar, filesystem_attachments, html5_notifier, jquery_mobile, jqueryui, libcalendaring, libkolab, managesieve, mobile, password, persistent_login, rcguard, show-folder-size_.

Can you tell me how to solve the problem?

Thank you.
With best regards
Nicola

**Comment by alecpl** [comment]
What happens with disabled all plugins?

**Comment by nsabatelli** [comment]
With all plugins disabled there is no error.

**Comment by nsabatelli** [comment]
I tested the configuration by disabling one plugin at a time and the error showed up in all cases.

It appears that only by disabling all plugins present in the configuration file (which are the following 'managesieve', 'password', 'calendar', 'html5_notifier', 'automatic_addressbook', 'mobile', 'persistent_login', 'rcguard', ' libkolab ',' show_folder_size '), the problem is solved.

**Comment by alecpl** [comment]
> I tested the configuration by disabling one plugin at a time and the error showed up in all cases.
> 
> It appears that only by disabling all plugins present in the configuration file (which are the following 'managesieve', 'password', 'calendar', 'html5_notifier', 'automatic_addressbook', 'mobile', 'persistent_login', 'rcguard', ' libkolab ',' show_folder_size '), the problem is solved.

You mean that using only core plugins you can reproduce this issue? Then maybe it has nothing to do with plugins and it is a coincidence that it worked with no plugins. Try again with only 'managesieve' and 'password' enabled.

What is your refresh/auto-save interval? Try with bigger value.

**Comment by nsabatelli** [comment]
Hello,
the problem has been solved.

Since my system consists of two servers in load balancing, I did some research and found that in this case it is necessary to activate the 'database_attachments' plugin to solve the problem I had posed.

Therefore, after making this change to the Roundcube configuration, the tests carried out by sending attachments were successful.

Thanks to the availability

Nicola
